### DISCUSSION TITLE
Cannot get rid of Warning: TNT will not destroy blocks above sea level.

### DISCUSSION THREAD

**Discussion by AlkonSI** [GriefPrevention/GriefPrevention#1988] [Q&A] [answered]
In the config, I set the value of TNT protection so it wouldn't be able to explode anyhwere, not only below sea level since I don't want people to grief. 

Now I have a really annoying problem, where this message will still pop. 
I've checked the config and saw nothing relevant, so I went in the messages file and first completly removed the said message, restarted server and it was still showing. After that I checked again and the message was back in the messages file. After that I tried just deleting the message part and leave it blank but that didn't work either. For last, I tried putting # infront of the said message but that didn't work either, the message still appears. 

What is there to do to fix this? Can anyone help very please.

**Comment by Jikoo** [comment, marked as answer]
You need to work with GP and Bukkit limitations to blank the message. You cannot remove the path from the config, the path must be there or GP will recreate it. Bukkit also reports a null value for a path the same way it does a nonexistent path. This means you need a quoted empty string.

Ex:
```yaml
valid-blank-nodes:
  my-blank-node: ''
  my-other-blank-node: ""
invalid-null-values:
  my-blank-node:
  # my-commented-out-node: "My value"
```
You must also restart your server to have changes to messages take effect. Messages are not reloaded by `/gpreload`.

**Reply by AlkonSI** [reply]
This worked, thank you sir!
